indexer/indexer_tests/categories_test.cpp

```diff
@@ -87,10 +87,7 @@ UNIT_TEST(LoadCategories)
 {
   classificator::Load();
 
-  CategoriesHolder h;
-  istringstream buffer(TEST_STRING);
-  h.LoadFromStream(buffer);
-
+  CategoriesHolder h(new MemReader(TEST_STRING, strlen(TEST_STRING)));
```

Comment:
OK, why not to change CategoriesHolder ctor from:
```c++
CategoriesHolder(Reader * p)
```
to
```c++
CategoriesHolder(unique_ptr<Reader> p)
```
?
It would be more readable and all questions about Reader lifetime will disappear.